File: app/src/main/java/jp/ac/dendai/c/jtp/adventuresample/scene/BadEnd.java
package jp.ac.dendai.c.jtp.adventuresample.scene;
import jp.ac.dendai.c.jtp.adventuresample.R;
public class BadEnd extends AbstractScene {
    @Override
    public GameState next(int no) {
        return null;
    }
    @Override
    public int getImageId() {
        return R.drawable.badend;
    }
    @Override
    public int getDateId(){
        return R.string.datebadend;
    }
    @Override
    public int getMessageId() {
        return R.array.messagebadend;
    }
    @Override
    public int getQuestionId() {
        return 0;
    }
    @Override
    public int getMusicId() { return R.raw.owarininokorumono; }
}

File: app/src/main/java/jp/ac/dendai/c/jtp/adventuresample/scene/BadEnd.kt
package jp.ac.dendai.c.jtp.adventuresample.scene

import jp.ac.dendai.c.jtp.adventuresample.R

class BadEnd : AbstractScene() {
    override fun next(no: Int): GameState? {
        return null
    }

    override val imageId= R.drawable.badend

    override val dateId = R.string.datebadend

    override val messageId = R.array.messagebadend

    override val questionId = 0

    override val musicId = R.raw.owarininokorumono
}